user-manual/transfer/import-metadata.rst

```diff
@@ -263,6 +264,199 @@ metadata. The JSON file is added to the metadata directory as above.
 
 The METS file will be populated exactly as in the CSV examples above.
 
+
+.. _metadata-xml-validation:
+
+Importing and validating descriptive metadata with source-metadata.csv
+----------------------------------------------------------------------
+
+TODO raison d'être paragraph.
```

Comment:
Users can add metadata files in xml format to their transfer, and have that metadata related to objects in the transfer and parsed into the METS file in the AIP. If formatted correctly, Archivematica can also validate the metadata to ensure it conforms with a local or linked standard.